<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExDateTime</class>
 <widget class="QWidget" name="ExDateTime">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>843</width>
    <height>333</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ExDateTime</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_7">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>日期时间（QTime、QDate、QDateTime）</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0" colspan="2">
         <widget class="QPushButton" name="btnGetDateTime">
          <property name="text">
           <string>获取当前时间日期</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="labDateTime">
          <property name="text">
           <string>日期时间</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="labDate">
          <property name="text">
           <string>日  期</string>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QLineEdit" name="editTime"/>
        </item>
        <item row="3" column="2">
         <widget class="QLineEdit" name="editDate"/>
        </item>
        <item row="4" column="2">
         <widget class="QLineEdit" name="editDateTime"/>
        </item>
        <item row="0" column="2" colspan="2">
         <widget class="QLabel" name="labCurrDataTime">
          <property name="text">
           <string>当前时间：</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="labTime">
          <property name="text">
           <string>时  间</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QTimeEdit" name="timeEdit">
          <property name="minimumSize">
           <size>
            <width>150</width>
            <height>0</height>
           </size>
          </property>
          <property name="calendarPopup">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QDateEdit" name="dateEdit">
          <property name="displayFormat">
           <string>yyyy年MM月dd日</string>
          </property>
          <property name="calendarPopup">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QDateTimeEdit" name="dateTimeEdit"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1" rowspan="2">
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>日历组件（Calendar Widget）</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_6">
      <item row="0" column="0">
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="0" column="0">
         <widget class="QLabel" name="labChoose">
          <property name="text">
           <string>选择的日期：</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="editChoose"/>
        </item>
        <item row="1" column="0" colspan="2">
         <widget class="QCalendarWidget" name="calendarWidget"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>定时器（QTimer）</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="0">
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="1" column="0">
         <widget class="QLabel" name="labPeriod">
          <property name="text">
           <string>定时周期</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QSpinBox" name="spinBox">
          <property name="suffix">
           <string>ms</string>
          </property>
          <property name="prefix">
           <string/>
          </property>
          <property name="maximum">
           <number>9999999</number>
          </property>
          <property name="value">
           <number>1000</number>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QLCDNumber" name="lcdSS"/>
        </item>
        <item row="0" column="2">
         <widget class="QLabel" name="labGo">
          <property name="text">
           <string>时间已流逝：</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLCDNumber" name="lcdmm"/>
        </item>
        <item row="2" column="0">
         <widget class="QLCDNumber" name="lcdHH"/>
        </item>
        <item row="1" column="2">
         <widget class="QPushButton" name="btnPeriod">
          <property name="text">
           <string>设置周期</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QPushButton" name="btnStatrt">
          <property name="text">
           <string>开始</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="btnStop">
          <property name="text">
           <string>停止</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0" colspan="3">
         <widget class="QProgressBar" name="progressBar">
          <property name="value">
           <number>3</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
